src/main/java/org/jitsi/videobridge/influxdb/LoggingHandler.java

```diff
@@ -306,71 +309,59 @@ public LoggingHandler(ConfigurationService cfg)
     @SuppressWarnings("unchecked")
     protected void logEvent(InfluxDBEvent e)
     {
-        // The following is a sample JSON message in the format used by InfluxDB
-        //  [
-        //    {
-        //     "name": "series_name",
-        //     "columns": ["column1", "column2"],
-        //     "points": [
-        //           ["value1", 1234],
-        //           ["value2", 5678]
-        //          ]
-        //    }
-        //  ]
-
-        boolean useLocalTime = e.useLocalTime();
-        long now = System.currentTimeMillis();
-        boolean multipoint = false;
-        int pointCount = 1;
-        JSONArray columns = new JSONArray();
-        JSONArray points = new JSONArray();
-        Object[] values = e.getValues();
-
-        if (useLocalTime)
-            columns.add("time");
-        Collections.addAll(columns, e.getColumns());
-
-        if (values[0] instanceof Object[])
-        {
-            multipoint = true;
-            pointCount = values.length;
-        }
-
-        if (multipoint)
-        {
-            for (int i = 0; i < pointCount; i++)
+        /* The following is a sample JSON message in the format used by InfluxDB v0.9.0
```

Comment:
Technically, this JSON format is deprecated as of 0.9.2, so next time we upgrade, we'll likely have to switch to their new custom text-based format for data submissions.